micro-rdk/src/common/registry.rs

```diff
@@ -24,6 +24,8 @@ use crate::proto::common::v1::ResourceName;
 pub enum RegistryError {
     #[error("RegistryError : Model '{0}' not found")]
     ModelNotFound(String),
+    #[error("RegistryError : Component Type '{0}' not found")]
+    ComponentTypeNotFound(String),
```

Comment:
what would be the difference between ModelNotFound and ComponentTypeNotFound

Reply:
Component type => "board", "motor", "sensor", etc.
Model type => "rdk:builtin:gpio_motor", "rdk:builtin:esp32", etc.